src/main/java/com/jcabi/w3c/DefaultValidationResponse.java

```diff
@@ -144,7 +148,9 @@ public Charset charset() {
      * @param error The error to add
      */
     public void addError(final Defect error) {
-        this.ierrors.add(error);
+        synchronized (this.ierrors) {
```

Comment:
@piddubnyi CopyOnWriteArraySet safe handles inserts 
we don't need synchronized block here